Add second test for LVM on RAID

vojtechtrefny opened the pull request:
This and the change to the first test case make sure the MD array can be referred by both its name and the "mount point" when used under a VG and also make sure the preferred name is not changed by or pykickstart.

This needs new version of both pykickstart and anaconda. See https://github.com/rhinstaller/anaconda/pull/6185 and https://github.com/pykickstart/pykickstart/pull/534

vojtechtrefny commented:
Marking as ready for review, new versions of both pykickstart and anaconda are now released and either already available in rawhide (pykickstart) or on its way to rawhide (anaconda).

rvykydal commented:
/test-os-variants

rvykydal reviewed (commented):
A typo on the commit message:
"... and also make sure the preferred name is not changed
by or pykickstart."

How should we deal with failures on rhel-9 and rhel-10 (/test-os-variants)?
- disable them with skip-on-rhel
- disable them with a reference to an jira or github issue and update the disabled tests table?, and reenable when they hit rhel composes?


Maybe it would be useful, if it is doable, to add a short description of the test at the beginning of the ks.in file (we do it for some of the tests) from which - reading it for both tests - would be clear what is specific for each of the test. (Way of referring to raid from volgroup command?)

Other than that looks good to me, thank you!

vojtechtrefny commented:
> A typo on the commit message: "... and also make sure the preferred name is not changed by or pykickstart."
> 

Fixed.

> How should we deal with failures on rhel-9 and rhel-10 (/test-os-variants)?
> 
>     * disable them with skip-on-rhel
> 
>     * disable them with a reference to an jira or github issue and update the disabled tests table?, and reenable when they hit rhel composes?
> 

I always forget the same tests are running both on Fedora and RHEL. The pykickstart bug is planned to be fixed in both RHEL 9 and 10 so I added these two test cases to skip-testtypes with the Jira issues referenced.

> 
> Maybe it would be useful, if it is doable, to add a short description of the test at the beginning of the ks.in file (we do it for some of the tests) from which - reading it for both tests - would be clear what is specific for each of the test. (Way of referring to raid from volgroup command?)
> 

Done.

rvykydal commented:
/test-os-variants

rvykydal commented:
> /test-os-variants

https://github.com/rhinstaller/kickstart-tests/actions/runs/14614842369/job/41000628075
Unfortunately, all raid tests are broken on rawhide / daily-iso currently: https://github.com/rhinstaller/kickstart-tests/issues/1414

rvykydal reviewed (approved):
Looks great to me, thank you.

vojtechtrefny commented:
Rebased to the latest main.

rvykydal commented:
/test-os-variants

rvykydal commented:
/test-tmt
